src/host/_stream.cpp

```diff
@@ -841,7 +543,7 @@ using Microsoft::Console::VirtualTerminal::StateMachine;
                       pcbBuffer,
                       nullptr,
                       textBuffer.GetCursor().GetPosition().x,
-                      WC_LIMIT_BACKSPACE,
+                      0,

```

Comment:
Why?

Reply:
It's kind of funny... Backspace in regular VT mode (and usual console mode) just means "move the cursor one column backwards, but that's it". There's no line wrapping or anything else. It's just backwards cursor movement.

`WC_DESTRUCTIVE_BACKSPACE` used to control whether this backwards movement overwrote the characters with whitespace. I renamed that flag to `WC_INTERACTIVE` because "destructive backspace" describes the implementation and not the wanted behavior. This is especially so since `WC_DESTRUCTIVE_BACKSPACE` always implied that `WC_LIMIT_BACKSPACE` was _not_ set. It's an interactive prompt after all, so backspacing needs to be able to wrap lines back up.

`WC_LIMIT_BACKSPACE` was used everywhere where `WC_DESTRUCTIVE_BACKSPACE` wasn't used.

This turned these two flags basically redundant and as implying the opposites. The new `WC_INTERACTIVE` combines them into one and now better describes the actual intention of the caller. And so this spot now gets a `0` because it's non-interactive, whereas all the `cmdline.cpp` parts get `WC_INTERACTIVE`.